examples/examples/journald.rs

```diff
@@ -10,7 +11,17 @@ fn main() {
         .with(tracing_subscriber::fmt::subscriber().with_target(false));
     match tracing_journald::subscriber() {
         Ok(subscriber) => {
-            registry.with(subscriber).init();
+            registry
+                .with(
+                    subscriber
+                        // We can tweak the mappings between the trace level and
+                        // the journal priorities.
+                        .with_priority_mappings(PriorityMappings {
+                            info: Priority::Informational,
+                            ..PriorityMappings::new()
+                        }),
+                )
+                .init();
```

Comment:
I think this example ought to be in the RustDoc documentation for the `with_priority_mappings` method, rather than in the general example file.